MetroscopeModelingLibrary/MultiFluid/HeatExchangers/Fogging.mo

```diff
@@ -0,0 +1,363 @@
+within MetroscopeModelingLibrary.MultiFluid.HeatExchangers;
+model Fogging
+  package FlueGasesMedium =
+      MetroscopeModelingLibrary.Utilities.Media.FlueGasesMedium;
+  package WaterSteamMedium =
+      MetroscopeModelingLibrary.Utilities.Media.WaterSteamMedium;
+  import MetroscopeModelingLibrary.Utilities.Units;
+  import MetroscopeModelingLibrary.Utilities.Units.Inputs;
+
+  // Boundary Conditions
+  Inputs.InputMassFlowRate Q_fg_in(start=Q_fg_in_0) "Inlet fg mass flow rate at the inlet";
```

Comment:
Sorry I had not realized that : don't ass `_in` to flows, since it is the same at the inlet and outlet. Also the comment is redundant
```suggestion
  Inputs.InputMassFlowRate Q_fg(start=Q_fg_0) "Inlet flue gases mass flow rate";
```
Note that you will have to update it everywhere in the file